samples/audio-wav/Makefile

```diff
@@ -6,8 +6,17 @@ INTDIR    	:= $(PROJDIR)/x64/Debug
 LIBS 		:= -lc -lkernel -lSceAudioOut -lSceUserService
 
 # Compiler options. You likely won't need to touch these.
-CC 			:= clang++
-LD 			:= ld.lld
+UNAME_S := $(shell uname -s)
+ifeq ($(UNAME_S),Linux)
+		CC  	:= clang++
+		LD 		:= ld.lld
+		CDIR	:= linux
+endif
+ifeq ($(UNAME_S),Darwin)
+		CC 		:= /usr/local/opt/llvm/bin/clang++
```

Comment:
this can be removed if you add `/usr/local/opt/llvm/bin/` to your PATH, then you wont need to include them in your if statement.

Reply:
Apple's LLVM is already on PATH, but it doesn't work with the toolchain. People can also add pure LLVM to path, but this would work whenever they choose to replace the Apple one in PATH or not